.github/workflows/release-cli.yml

```
@@ -216,61 +219,23 @@ jobs:
-
-          **Download URLs:**
-          $(jq -r '.["cli-packages"][]' package.json | sed "s|^|- https://github.com/raghavyuva/nixopus/releases/download/nixopus-$CLI_VERSION/|")
-
-          This PR was automatically created by the CLI release workflow." \
-            --base master \
-            --head "$BRANCH_NAME" || echo "Failed to create PR, but continuing with release"
-            
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        if: ${{ needs.build-and-package.outputs.version != '' }}
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.7)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)